src/test/java/core/basesyntax/strategy/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandlerService;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseHandlerTest {
+    private OperationHandlerService operationHandlerService;
+    
+    @BeforeEach
+    void setUp() {
+        operationHandlerService = new PurchaseHandler();
+        Storage.fruitsStorage.clear();
+    }
+
+    @Test
+    void purchaseOperationValidAmount_NotOk() {

```

Comment:
the word after underscore should start from lower case letter 
fix it in all tests 
```suggestion
    void purchaseOperationValidAmount_notOk() {

```